tests/test_spatial_tools.py

```diff
@@ -182,7 +210,27 @@ def make_gdal_hillshade(filepath) -> np.ndarray:
 
 class TestRobustFitting:
 
-    def test_robust_polynomial_fit(self):
+    @pytest.mark.parametrize("pkg_estimator", [('sklearn','Linear'), ('scipy','Linear'), ('sklearn','Theil-Sen'),
+                                           ('sklearn','RANSAC'),('sklearn','Huber')])
+    def test_robust_polynomial_fit(self, pkg_estimator: str) -> None:
+
+        np.random.seed(42)
+
+        # x vector
+        x = np.linspace(1, 10, 1000)
+        # exact polynomial
+        true_coefs = [-100, 5, 3, 2]
+        y = true_coefs[0] + true_coefs[1] * x + true_coefs[2] * x ** 2 + true_coefs[3] * x ** 3
+
+        coefs, deg = xdem.spatial_tools.robust_polynomial_fit(x, y, linear_pkg=pkg_estimator[0], estimator=pkg_estimator[1], random_state=42)
+
+        assert deg == 3 or deg == 4
+        assert np.abs(coefs[0] - true_coefs[0]) <= 100
+        assert np.abs(coefs[1] - true_coefs[1]) < 5
```

Comment:
Could also do `assert coefs[0] == pytest.approx(true_coeffs[0], abs=100)` 

Don't know if it's actually more readable now that I wrote it...

Reply:
It is clearer! Done in 5cfe2e7